spectrum/autism_prevalence_map/management/commands/pull_studies_from_gsheets.py

```diff
@@ -25,11 +26,13 @@ def load_research_data(self):
         source = json.loads(trimmed_response)
         print(source['table']['cols'])
 
-        for data in source['table']['rows']:
-            # print(data['c'])
+        # delete all the rows
+        studies.objects.all().delete()
+
+        for index, data in enumerate(source['table']['rows']):
             try:
                 #skip if year not a date
-                print(data['c'][0]['f'] + ', ' + data['c'][1]['v'])
+                # print(data['c'][0]['f'] + ', ' + data['c'][1]['v'])
```

Comment:
Is there any reason you're commenting this line out instead of deleting it?

Reply:
I commented it out because I thought we might want to check that later, but I can also delete it.